LunaMary/src/TypeQuiz.java

```diff
@@ -0,0 +1,18 @@
+public class TypeQuiz {
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    String name;
```

Comment:
please put attributes before methods, is a good convention. Also please add the access modifier which will prevent further errors.